core/spec.md

```diff
@@ -5252,33 +5415,26 @@ as the semantic minor version identifier.
 #### Default Version of a Resource
 
 As Versions of a Resource are added or removed there needs to be a mechanism
-by which the "default" one is determined. There are three options for how this
+by which the "default" one is determined. There are two options for how this
 might be done:
 
-1. Server's choice. If the Resource's model has the `setdefaultversionsticky`
-   aspect set to `false` then the server will always choose which Version is
-   the "default" and any attempt by the client to control it MUST result in the
-   generation of an error.
-
-   This specification does not mandate the algorithm that the server uses,
-   however the default choice SHOULD be "newest = default" (option 2).
-
-2. Newest = Default. The newest Version created (based on `createdat` timestamp)
+1. Newest = Default. The newest Version created (based on `createdat` timestamp)
    is always the "default" Version. This is the default choice. If more than
    one Version has the same "newest" `createdat` timestamp, then the choice
    the Version with the highest alphabetical `versionid` value using case
    insensitive compares MUST be chosen. For example, if there are 3 Versions
    with `versionid` values of `v10`, `z1` and `V2`, then ordering of the
    Versions from oldest to newest would be: `v10`, `V2`, `z1.
 
-3. Client explicitly chooses the "default". In this option, a client has
+2. Client explicitly chooses the "default". In this option, a client has
```

Comment:
Suggest using "1" for all numbers in numbered lists, and Markdown will just do the right thing. (I believe, anyway. You *might* need to remove the blank line above.)